cmd/describeaccount_test.go

```diff
@@ -166,3 +167,20 @@ func TestDescribeAccount_Latency(t *testing.T) {
 	require.NoError(t, err)
 	require.Contains(t, out, "lat (10%)")
 }
+
+func TestDescribeAccount_Json(t *testing.T) {
+	ts := NewTestStore(t, "O")
+	defer ts.Done(t)
+
+	ts.AddAccount(t, "A")
+	out, _, err := ExecuteCmd(rootCmd, "describe", "account", "--json")
+	// reset the global
+	Json = false
+	require.NoError(t, err)
+	m := make(map[string]interface{})
+	err = json.Unmarshal([]byte(out), &m)
+	require.NoError(t, err)
+	ac, err := ts.Store.ReadAccountClaim("A")
+	require.NoError(t, err)
+	require.Equal(t, ac.Subject, m["sub"])
```

Comment:
In other tests you would have the equivalent of `require.NotNil(t, ac)` here..

Reply:
I did it because the IDE was complaining that the value could be null - but it isn't. - the API returns null/empty values on error